src/wolfio.c

```diff
@@ -388,7 +388,7 @@ static int sockAddrEqual(
 #ifndef WOLFSSL_IPV6
 static int PeerIsIpv6(const SOCKADDR_S *peer, XSOCKLENT len)
 {
-    if (len < sizeof(peer->ss_family))
+    if (len < (int)sizeof(peer->ss_family))
```

Comment:
Should this be cast to a `(XSOCKLENT)`, since that is the type of `len`?

Reply:
Yes, that is true. Fixed.